flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -53,6 +53,14 @@ if [ $EXIT_CODE == 0 ]; then
     EXIT_CODE=$?
 fi
 
+if [ $EXIT_CODE == 0 ]; then
+  printf "\n==============================================================================\n"
+  printf "Running Quickstarts nightly end-to-end test\n"
```

Comment:
this test should only be run nightly, i.e. must not show up in this file.

Reply:
Yes. I also feel strange put this in this file, but I watched the same like ```test_streaming_kafka010.sh``` in this file. And this confused me. I do not think ```test_streaming_kafka010.sh``` should put here since ```kafka-common.sh``` will execute a download also.